bundle/deploy/terraform/plan.go

```diff
@@ -59,7 +110,7 @@ func (p *plan) Apply(ctx context.Context, b *bundle.Bundle) error {
 		IsEmpty:      !notEmpty,
 	}
 
-	cmdio.LogString(ctx, fmt.Sprintf("Planning complete and persisted at %s\n", planPath))
+	cmdio.LogString(ctx, "Planning complete")
```

Comment:
This doesn't seem relevant to me as a user?

If we want to give some sense of progress we could show a spinner. But otherwise I would just to show anything here.

Reply:
Removed this. Good point btw.